rpi_csdt_community/templates/rpi_csdt_community/classrooms_guide.html

```diff
@@ -0,0 +1,41 @@
+            <h1>CSDT Classroom Creation Guide</h1>
+            <p>
+                Thank you for your interest in signing your classroom up to use our software, Culturally Situated Design Tools. We have programmed a "classrooms" app on our website to facilitate teachers who are running a classroom of students, to address their needs as facilitators. CSDTs are set up to simulate a cloud computing experience. Each user can save their program to the cloud, which shows up under their profile even if they're not in a classroom. The purpose of classrooms is to organize this cloud computing into a sort of gallery or boutique for all the students in the classroom, including the teacher. With a classroom, students can create and save their programs, then submit them for approval by the teacher or teachers. The teacher approves projects using a visual representation of the program, to verify that it is school appropriate, and then all approved projects are added to the pool of works created by the classroom. This pool of shared works can be viewed by other students, commented on, and even remixed by other students using the other's person's source code. Classrooms allow for sharing between students and at-a-glance observation of who is creating what, providing motivation and teamwork. For the teacher, it permits an easy source to view who is keeping up with the tutorials, who is struggling, who is completing assignments, who is succeeding, and who is creating genuinely interesting works of art.
+            </p>
+            <p>
+                CSDTs use code to create amazing pieces of visual art that can elevate programming tutorials into a completely different realm of experience. Too often programming tutorials are just Input/Output exercises, using if statements to return a variety of text replies based on the range of an input number, for example. Not to say those types of tutorials aren't important, but CSDTs let your students make colorful, animated, visually engrossing pieces that can be shared with others in your class. The Classrooms app lets your students share what they make, so that everyone can see what range of colors inspires whom and learn collectively to make better and better programmable art. It turns programming into a discussion group, a sort of forum where students can peek inside other project's code to see how things are created, then adapt that code for their own projects. I see classrooms as facilitating a sort of "critique time" where every student hangs their art on the wall, and the class as a whole gets to critique each piece one by one and discuss their emotional reactions to the piece. Just as this helps a developing artist, the same is true for beginning programmers. Our CSDT programs are designed to facilitate focus on the cultural aspects of programming, and to turn your computer lab into an art studio through the Classrooms app goes beyond the normal experiences of introductory programming courses.
```

Comment:
Delete, they don't need to know why you like CSDTs... this is a guide for class creation.